docs/intro.md

```diff
@@ -159,7 +159,19 @@ Routes are defined using a configuration file or by POSTing directly to the appl
 
 ![simple-add-rig](assets/simple-add-rig.png)
 
-## Next Layer of Complexity: A Reference Architecture
-In the [/examples](https://github.com/Accenture/reactive-interaction-gateway/tree/master/examples) folder, there is an example architecture made with a React frontend, RIG, a Node backend, and a Kafka instance. Here's a chart demonstrating that reference architecture in abstract. Go to [/examples](https://github.com/Accenture/reactive-interaction-gateway/tree/master/examples) for more depth:
+## Another Architectural Option: Using RIG for Publish / Subscribe Communication with the Backend
+It is possible to send an HTTP request to RIG and have it publish and subscribe to Kafka, decoupling the front end from the backend entirely.
```

Comment:
I once had a discussion around that with one of our technical writers that review our official blog posts. The proper way to do it "frontend" and "backend", but "front-end service" and "back-end service". So if it's used as a stand-alone noun, it is one word.

Reply:
Good clarification. And very interesting, I wonder why the hyphenation in one case and not the other? Something about being an adjective rather than a noun? I will update accordingly.

Reply:
I just looked up her explanation:

> I see back end, backend and back-end used interchangeably in all sorts of contexts, technical or not. You have four instances of backend and four of frontend in your post. The words should be hyphenated if used as a compound adjective in front of another word. For example, you should have back-end services and back-end team but not HTTP backends. Same for front-end team and front-end code but not frontends or users’ frontends.